### PR TITLE
Beat inclusion

### PR DESCRIPTION
**Pull request by hvasbath**
Added KAUST and Uni Potsdam Logos, panel for BEAT and a new reference

### CODE REVIEW

File: source/index.rst [hunk 1]
```diff
@@ -269,6 +271,45 @@ estimation problems.
...
+
+.. image:: _static/mt_correlation.png
+    :align: left
+
+Flexible, bayesian, multi-dataset deformation source optimization including model uncertainties.
+Probabilistic source inversion for point sources, extended sources up to distributed slip.
+
+* convergence insurance through Sequential Monte Carlo algorithm
+* option to include model uncertainties (theory errors due to Earth structure)
+* parallel sampling on n CPUs
```
Comment (miili) [inline source/index.rst]: available CPUs

File: source/index.rst [hunk 2]
```diff
@@ -26,6 +26,8 @@ Development and support is coordinated at https://github.com/pyrocko.
        <a href="https://www.bridges.uni-kiel.de/"><img style="margin-left:1em; height:2em; display:inline-block;" src="_static/logos/bridges.svg" /></a>
        <a href="http://www.uni-kiel.de/"><img style="margin-left:1em; height:2em; display:inline-block;" src="_static/logos/cau.svg" /></a>
        <a href="http://www.uni-hamburg.de/"><img style="margin-left:1em; height:2em; display:inline-block;" src="_static/logos/uhh.svg" /></a>
+       <a href="http://www.uni-potsdam.de/"><img style="margin-left:1em; height:2em; display:inline-block;" src="_static/logos/up.svg" /></a>
```
Comment (miili) [inline source/index.rst]: I would prefer this elongated logo
https://publishup.uni-potsdam.de/layouts/pmatrix/img/logo/unilogo.png

File: source/index.rst [hunk 3]
```diff
@@ -26,6 +26,8 @@ Development and support is coordinated at https://github.com/pyrocko.
...
+       <a href="http://www.kaust.edu.sa/"><img style="margin-left:1em; height:2em; display:inline-block;" src="_static/logos/kaust.svg" /></a>
```
Comment (miili) [inline source/index.rst]: Is it possible to use this KAUST logo?
https://orcid.org/sites/default/files/ckfinder/userfiles/images/KAUST%20Logo.png

File: source/index.rst (hunk 2 above)
Comment (hvasbath) [reply source/index.rst]: ok just need to find the svg version ...

File: source/index.rst (hunk 3 above)
Comment (hvasbath) [reply source/index.rst]: maybe its internationally better your right, also need to find SVG

**Comment by hvasbath** [comment]
added image files and changed text

**Pull request merged**